lib/routes/novel/biquge.js

```diff
@@ -3,7 +3,7 @@ const cheerio = require('cheerio');
 const iconv = require('iconv-lite');
 const asyncPool = require('tiny-async-pool');
 
-const baseUrl = 'https://www.biquge5200.com/';
+const baseUrl = 'https://www.biquge5200.net/';
```

Comment:
```suggestion
const baseUrl = 'http://www.biquge5200.net/';
```
The site does not have HTTPS enabled.